src/Common/tests/Configurations.props

```diff
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
       netstandard1.3-Unix;
```

Comment:
For the test projects I think we should go a head and remove all the non-version-less configurations. We might need them for some of the src/ref projects but we don't need them for the test projects.